index.test-d.ts

```diff
@@ -8,6 +8,38 @@ import {
 	ExecaSyncError
 } from '.';
 
+try {
```

Comment:
All those properties are already tested, i.e. don't need to be tested again.

The only properties that need testing are the options of `kill()` (and those are not tested at the moment).

Reply:
Like this?: b85310f5626281c7c9bbb2352281e104a824778b

I have to admit that I don't really understand this kind of testing (if that's what you want to see).
Is this only for semver checking basically?

Reply:
You can find out more about this kind of TypeScript tests here: https://github.com/SamVerschueren/tsd